src/gui/tray/usermodel.h

```diff
@@ -125,6 +125,7 @@ public slots:
 
     QElapsedTimer _guiLogTimer;
     NotificationCache _notificationCache;
+    QMimeDatabase mimeDb;
```

Comment:
Please add underscore prefix as in other private members

Reply:
Added, thanks